packages/react-components/react-accordion/src/components/AccordionPanel/AccordionPanel.tsx

```diff
@@ -4,6 +4,7 @@ import { renderAccordionPanel_unstable } from './renderAccordionPanel';
 import { useAccordionPanelStyles_unstable } from './useAccordionPanelStyles';
 import type { AccordionPanelProps } from './AccordionPanel.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
```

Comment:
_nit:_ Keep `import type` statements at the bottom

```suggestion
import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
import { useAccordionPanelStyles_unstable } from './useAccordionPanelStyles';
import type { AccordionPanelProps } from './AccordionPanel.types';
import type { ForwardRefComponent } from '@fluentui/react-utilities';
```